packages/DataGrid/src/DataGrid.js

```diff
@@ -0,0 +1,628 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { VariableSizeGrid as Grid } from "react-window";
+import extractChildren from "@paprika/helpers/lib/extractChildren";
+import Spinner from "@paprika/spinner";
+import Bubbly from "@paprika/icon/lib/BubblySearchNoResults";
+import Cell from "./components/Cell";
+import useGridEventHandler from "./hooks/useGridEventHandler";
+import ColumnDefinition from "./components/ColumnDefinition";
+import * as styled from "./DataGrid.styles";
+import WhenScrollBarReachedBottom, { End } from "./components/WhenScrollBarReachedBottom";
+import InfinityScroll from "./components/InfinityScroll";
+
+const propTypes = {
+  children: PropTypes.node.isRequired,
+  data: PropTypes.arrayOf(PropTypes.shape({})),
```

Comment:
No shape defined? So just an array of `object`s?

Reply:
yeah, is the beauty the table is data agnostic the only thing that needs is an array of objects, where the objects are the rows and the attributes on those objects the columns